databricks_cli/sdk/service.py

```diff
@@ -170,16 +238,26 @@ def run_now(self, job_id=None, jar_params=None, notebook_params=None, python_par
             _data['notebook_params'] = notebook_params
         if python_params is not None:
             _data['python_params'] = python_params
-        if python_named_params is not None:
-            _data['python_named_params'] = python_named_params
```

Comment:
Why we need to move it? having python params close to each other is easier to read